config/routes.rb

```diff
@@ -15,6 +15,7 @@
         resources :columns, only: [:index]
       end
       get '/get_all_nodes', to: 'nodes#get_all_nodes'
+      resources :newsletter_subscriptions, only: [:create]
```

Comment:
maybe don't remove it from v2 just yet, so that we don't break anything in the frontend?

Reply:
Yes, I know. I didn't remove it. The routes were duplicated, so I just removed one of them.
It still exists in line 24.